### ISSUE TITLE
Fatal error

### ISSUE THREAD

**Issue by trevor-td** [Bannerets/tdl#119]
Hi,

Apologies if this has already been addressed, but I have the following issue after the tdl client has been running for a while.  This is from a little while ago and am in the process of trying to reproduce the error.
![unknown](https://user-images.githubusercontent.com/109141394/180977550-511ebb1c-f17d-4d92-a49a-c2c9c988945b.png)

This error occurred while using Node 16.  I had found a another reply on another forum where they had experienced a similar issue and resolved it by downgrading to Node 12.  There is a requirement for me to use Node 16 however, is there any way around this?

Thanks in advance.

**Comment by Bannerets** [comment]
Hi! Thanks for reporting. Could you send the full stack trace?

This is related to https://github.com/nodejs/node/issues/32463 and https://github.com/Bannerets/tdl/issues/80

**Comment by Bannerets** [comment]
I reproduced the error on Github Actions:

```
#
# Fatal error in , line 0
# Check failed: result.second.
#
#
#
#FailureMessage Object: 0x7ffdea674aa0
 1: 0xb7dd11  [node]
 2: 0x1c15754 V8_Fatal(char const*, ...) [node]
 3: 0x100cd11 v8::internal::GlobalBackingStoreRegistry::Register(std::shared_ptr<v8::internal::BackingStore>) [node]
 4: 0xd2[41](https://github.com/Bannerets/tdl/runs/7521391301?check_suite_focus=true#step:7:42)e8 v8::ArrayBuffer::GetBackingStore() [node]
 5: 0xacaf4c napi_get_arraybuffer_info [node]
 6: 0x7f5a94922870 td_client_receive(Napi::CallbackInfo const&) [/home/runner/work/tdl/tdl/packages/tdl-tdlib-addon/build/Release/td.node]
 7: 0x7f5a9492a22f Napi::details::CallbackData<void (*)(Napi::CallbackInfo const&), void>::Wrapper(napi_env__*, napi_callback_info__*) [/home/runner/work/tdl/tdl/packages/tdl-tdlib-addon/build/Release/td.node]
 8: 0xac220f  [node]
 9: 0xd6014b  [node]
10: 0xd615ec  [node]
11: 0xd61c66 v8::internal::Builtin_HandleApiCall(int, unsigned long*, v8::internal::Isolate*) [node]
12: 0x160c699  [node]
Illegal instruction (core dumped)
```

**Comment by Bannerets** [comment]
This should be fixed in 61211eaf730a47f8f075c552d4cb4befbf48cf06. I've released `tdl-tdlib-addon@1.1.0` with the fix. Thanks for reporting again.

**Comment by trevor-td** [comment]
Thank you @Bannerets for all your efforts.  Much appreciated.
